content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: XXXX'
+date: 2021-04-08
+slug: kubernetes-1-21-release-announcement
+---
+
+**Authors:** [Kubernetes 1.21 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release_team.md)
+
+We’re pleased to announce the release of Kubernetes 1.21, our first release of 2021!
+This release consists of 50 enhancements: 15 enhancements have graduated to stable, 15 enhancements are moving to beta, and 19 enhancements are entering alpha.
+
+## Major Themes
+
+### CronJobs Graduate to Stable!
+CronJobs (previously ScheduledJobs) has been a beta feature since Kubernetes 1.8! With 1.21 we get to finally see this widely used API graduate to stable.
+
+CronJobs are meant for performing all time-related actions, namely backups, report generation and the like. Each of these tasks should be allowed to run repeatedly (once a day/month, etc.) or once at a given point in time.
+https://github.com/kubernetes/enhancements/issues/19
+
+### IPv4/IPv6 dual-stack support
+IP addresses are a consumable resource operators and cluster administrators need to make sure are not exhausted. Having dual-stack support enables native IPv6 routing to pods and services, and it also improves a possible scaling limitation for workloads.
+
+Dual-stack support in Kubernetes means pods, services, and nodes can get IPv4 addresses and IPv6 addresses. In Kubernetes 1.21 dual-stack support is graduating from alpha to beta and is now enabled by default.
+
+### Immutable Secrets and ConfigMaps
+Immutable Secrets and ConfigMaps adds a new field to those resource types which will reject changes to those objects if set. Secrets and ConfigMaps by default are mutable which is beneficial for pods that are able to consume changes. Mutating Secrets and ConfigMaps can also cause problems if a bad configuration is pushed for pods that use them.
+
```

Comment:
If it's useful, we can link to [`/docs/concepts/configuration/configmap/#configmap-immutable`](https://kubernetes.io/docs/concepts/configuration/configmap/#configmap-immutable) and [`/docs/concepts/configuration/secret/#secret-immutable`](https://kubernetes.io/docs/concepts/configuration/secret/#secret-immutable)

Reply:
I've added this in the latest commit, believe it would be useful. Thank you!